test/helpers.js

```
@@ -35,3 +86,462 @@ export async function exec(cmd, ...args) {
+    throw new Error(
+      "JSPI async type skipped, but JSPI was not enabled -- please ensure test is run from an environment with JSPI integration (ex. node with the --experimental-wasm-jspi flag)",
+    );
+  }
+
+  // Build a directory for the transpiled component output to be put in
+  // (possibly inside the passed in outputDir)
+  const moduleOutputDir = join(outputDir, component.name);
+  try {
+    await stat(moduleOutputDir);
+  } catch (err) {
+    if (err && err.code && err.code === "ENOENT") {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)